### PR TITLE
Added tests for "Master port 50826"

### PR DESCRIPTION
**Pull request by chotea**
Added tests for https://github.com/saltstack/salt/pull/57007
Did a rebase on Alexey's branch and added tests on top of it.

### CODE REVIEW

**Review by Akm0d — APPROVED** [review]

**Review by cmcmarrow — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

File: salt/modules/opkg.py [hunk 1]
```diff
@@ -103,21 +99,25 @@ def _get_restartcheck_result(errors):
     return rs_result
 
 
-def _process_restartcheck_result(rs_result):
+def _process_restartcheck_result(rs_result, **kwargs):
```
Comment (s0undt3ch) [inline salt/modules/opkg.py:102]: Why do we need to use `**kwargs` instead of `always_restart_services=True`?

File: salt/modules/opkg.py [hunk 2]
```diff
@@ -438,6 +438,9 @@ def install(
 
         .. versionadded:: 2017.7.0
 
+    always_restart_services
```
Comment (s0undt3ch) [inline salt/modules/opkg.py:441]: Why isn't this added to the `install` function as a keyword argument?

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

**Comment by garethgreenaway** [comment]
@chotea Looks like you've got some merge conflicts and some requests for changes from @s0undt3ch.  Mind swinging back on this one?

**Review by s0undt3ch — CHANGES_REQUESTED** [review]
Still think we might not need to use `**kwargs`

**Comment by dwoz** [comment]
Closing this due to inactivity. Anyone should feel free to re-open it if they want to see it through to the end in one release cycle.

**Pull request closed without merging**